### PR TITLE
Namespaces as global variables, fixed typo, added doc about json-ld

### PR DESCRIPTION
**Pull request by agbeltran**

### CODE REVIEW

File: data_management/prov.py [hunk 1]
```diff
@@ -643,28 +647,28 @@ def generate_prov_document(data_product, depth, request):
...
+        doc.add_namespace(reg_uri_prefix, central_registry_url)
     else:
         # we are using a local registry
         reg_uri_prefix = 'lreg'
         doc.add_namespace(reg_uri_prefix, url)
 
     # the vocab namespace is always the main registry
-    doc.add_namespace(FAIR_VOCAB_PREFIX, f'{cenral_registry_url}vocab/#')
+    doc.add_namespace(FAIR_VOCAB_PREFIX, f'{central_registry_url}vocab/#')
     # we need to tell SONAR to ignore 'http' in the vocab URLs
```
Comment (antony-wilson) [inline data_management/prov.py:667]: Move this comment to line 18

File: data_management/prov.py [hunk 2]
```diff
@@ -643,28 +647,28 @@ def generate_prov_document(data_product, depth, request):
...
-    doc.add_namespace(DCAT_VOCAB_PREFIX, 'http://www.w3.org/ns/dcat#')  # NOSONAR
-    doc.add_namespace(DCMITYPE_VOCAB_PREFIX, 'http://purl.org/dc/dcmitype/')  # NOSONAR
-    doc.add_namespace(DCTERMS_VOCAB_PREFIX, 'http://purl.org/dc/terms/')  # NOSONAR
-    doc.add_namespace(FOAF_VOCAB_PREFIX, 'http://xmlns.com/foaf/spec/#')  # NOSONAR
+    doc.add_namespace(DCAT_VOCAB_PREFIX, DCAT_VOCAB_NAMESPACE)  # NOSONAR
```
Comment (antony-wilson) [inline data_management/prov.py:668]: remove the `# NOSONAR` from these lines and add them to end of the *_NAMESPACE lines at the top of the file

**Comment by agbeltran** [comment]
the security issues highlighted need to be ignored because those are the vocabularies' namespaces

File: data_management/prov.py (hunk 2 above)
Comment (agbeltran) [reply data_management/prov.py:668]: ah, great, will do

**Review by antony-wilson — APPROVED** [review]

**Pull request merged**